src/components/App/App.js

```diff
@@ -31,24 +31,27 @@ export default class App extends React.Component {
       searchValue,
       onLogoClick,
       resultsValue,
-      loadingResults
+      loadingResults,
+      errorHandled
```

Comment:
if you want, you can group these props in a subobject `resultsPanelProps` to have a cleaner `jsx`
PS: see comment on `template`